lib/travis/logs/drain.rb

```diff
@@ -42,12 +42,12 @@ def run_loop_tick
           Travis.logger.debug('checking drain consumer', name: name)
           if consumer.dead?
             dead << name
-            Travis.logger.info('dead consumer found', name: name)
+            Travis.logger.debug('dead consumer found', name: name)
```

Comment:
are you seeing a lot of these? the reason i ask is that it can indicate issues with the rabbit connection.

Reply:
Oh interesting. Think this is something that should stay `info`? Looking at my test instance I def see a bunch of it. But also I'm stopping and starting the container a bunch so maybe that's part of it.

Reply:
> Think this is something that should stay info?

Clarifying: seems like it may be something that'd maybe help us diagnose if there were rabbit problems, etc. So maybe debug is burying it too far?

Reply:
just checked our production logs in papertrail and i'm not seeing any of these at all. so unless they are noisy without causing issues on enterprise i think there is value in keeping this one at `info`.

Reply:
Thanks for checking that! I have a hunch that something isn't configured correctly in the current Enterprise build (a lot changed with `travis-logs` since the last version and I wonder if I missed a spot). Still will probably switch this back to `info` since it gives us some info of the health of things and isn't simply a ping with low information.